distribution/docs/src/main/resources/content/_configuring/connecting-to-external-idp.adoc

```diff
@@ -7,13 +7,19 @@
 
 == {title}
 
-.Service Provider Metadata
-See <<{managing-prefix}configuring_idp_sp, Configuring Security IdP Service Provider>> for details on configuring the IdP client.
+To connect to an external Identity Provider,
+
+. Provide the external IdP with ${branding}'s Service Provider (SP) metadata.
+The SP metadata can found at `https://<FQDN>:<PORT>/services/saml/sso/metadata`.
 
-.Disable Included IdP
-Remove the included IdP.
+. Replace the IdP metadata field in ${branding}.
+.. Navigate to the *${admin-console}*.
+.. Select the *${ddf-security}* application.
+.. Select the *Configuration* tab.
+.. Select *IdP Client*.
+.. Populate the *IdP Metadata* field with the external IdP's metadata.
```

Comment:
I was going to add a note here that we may not interoperate with all IdPs and to use the SAML CTK to identify the once we can interoperate with. Not sure if I should. Thoughts?

Reply:
We should hold off until it's released.

Reply:
I don't have any objections to that, but we'll see what others say

Reply:
I'd love to see a reference/link to it after we've released it.